riak_test/opbcounter_test.erl

```diff
@@ -33,7 +37,7 @@ confirm() ->
     decrement_test(Nodes),
     transfer_test(Nodes),
     conditional_write_test(Nodes),
-    rt:clean_cluster(Nodes),
+    %rt:clean_cluster(Nodes),
```

Comment:
I can added, but it will slow down everything. I think a simple way to avoid cleaning the cluster is to use different keys in the tests. (we could use the module_name + counter). Just a suggestion.

Manuel

> On 14 Sep 2015, at 13:56, Annette Bieniusa <notifications@github.com> wrote:
> 
> In riak_test/opbcounter_test.erl <https://github.com/SyncFree/antidote/pull/143#discussion_r39389615>:
> 
> > @@ -33,7 +37,7 @@ confirm() ->
> >      decrement_test(Nodes),
> >      transfer_test(Nodes),
> >      conditional_write_test(Nodes),
> > -    rt:clean_cluster(Nodes),
> > +    %rt:clean_cluster(Nodes),
> I think we still need it - at the end of every independent test case. 
> Tests must be independent, and cleaning up the test fixture is essential for this.
> In the long run, someone will otherwise introduce "bugs" there.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SyncFree/antidote/pull/143/files#r39389615>.
>